### ISSUE TITLE
Open files do not close when closing a project

### ISSUE THREAD

**Issue by tribis** [Komodo/KomodoEdit#2049]
### Short Summary
When 
`Preferences > Workspace >when closing a project: "close all open files in project"`
is set, closing a project should close the open files, but it does not under certain conditions

### Steps to Reproduce
Add a new project, save the project definition inside, say:
`C:\Users\myname\Documents\myprojects\`

`Add > Existing folder..`
to the new project
Select a folder like: 
`C:\Users\myname\Documents\mycode\`
that is, *not  a subfolder of the above*
Save the project

Double click on the new folder and show the files and folders existing inside it
double click open one or more files
Right click on the project name and select "Close Project"
### Expected results
The project closes, hiding the added folder
The open files are closed

### Actual results
The project closes, hiding the added folder
The open files stay opened

### Platform Information
Komodo IDE, version 10.1.1, build 89474, platform win32-x86. Built on Thu Sep 01 14:27:01 2016.

**Comment by mitchell-as** [comment]
That's odd. I followed the steps and was not able to reproduce. All of my open files closed. @cgchoffman would you please test this on Windows? It might be a Windows-only bug.

**Comment by Naatan** [comment]
Please have a look at Preferences > Workspace > Opening and Closing Projects.

You can change the behavior there.

**Comment by mitchell-as** [comment]
@Naatan he tried that and said it does not work under certain conditions which he then outlined.

**Comment by cgchoffman** [comment]
He has that set.  He mentioned it in the forums.  i was able to easily reproduce this.

**Comment by cgchoffman** [comment]
The only thing I don't agree with and isn't how it has ever worked is your first step in the "expected" field.

> The project closes, hiding the added folder

 The Places pane directory won't change to anything until you tell it to, explicitly or implicitly through some other action.

**Comment by mitchell-as** [comment]
@cgchoffman I think he means the "Projects" pane. In my case, the added external folder did hide, but my places pane remained the same.

**Comment by tribis** [comment]
@cgchoffman  yes @mitchell-as  has understood correctly: when I close the project, the added folder disappears from the project pane (nothing essential there, however)

**Comment by Naatan** [comment]
That part is as expected though, no?

From my understanding on reading the ticket the only thing that's going "wrong" is your files are not closing?

**Comment by tribis** [comment]
Yes, that's correct. And as I see it, it is also corresponds to me writing:

> Expected results

>The project closes, hiding the added folder
>The open files are closed
>Actual results

>The project closes, hiding the added folder
>The open files stay opened

**Comment by Naatan** [comment]
Ok, thanks :)

@cgchoffman so you reproduced that the files did not close despite "Close all open files in the project" being set under Workspace prefs?

**Comment by cgchoffman** [comment]
Yes I can repro 100%.  None of the files from the added folder are closed.  Any files from the root directory of the project are closed.
